src/umrt_imu_interface/cpack.cmake

```
@@ -0,0 +1,33 @@
+set(MAINTAINER_NAME "Noah Reeder <noahreederatc@gmail.com>")
+set(PROJECT_VERSION_MAJOR 0)
+set(PROJECT_VERSION_MINOR 0)
+set(PROJECT_VERSION_PATCH 1)
+
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Library for interacting with the encoders on the University of Manitoba Robotics Team's robotic arm"
```

```suggestion
set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Library for interacting with the encoders on the University of Manitoba Robotics Team's drivetrain"
```

---

Whoops, got mixed up
```suggestion
set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Library for interacting with the IMU on the University of Manitoba Robotics Team's rover"
```